src/utilities/alerter.ts

```diff
@@ -4,14 +4,27 @@ import { IConfig } from '../types/IConfig';
 import { IDevice } from '../types/IDevice';
 import { arpscanDevicesAsync } from './scanner';
 
-interface IEmail {
-	text: string;
+const defaultAlertMessages = {
+	intruder:'The fortress is in peril.',
+	departure: 'Goodbye.',
+	arrival:'Welcome home.',
+	doorOpen:'The door is open.',
+	knownDevice:'A recognized device arrived.',
+}
+
+export type AlertType = keyof typeof defaultAlertMessages;

```

Comment:
🤔 I feel like there should be a way in Typescript to match these keys to the corresponding keys in `emailRecipients` in `exampleConfig.ts`.